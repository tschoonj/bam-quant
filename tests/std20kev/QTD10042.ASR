File: C:\xrfdaten\2014KW30\chris\std20kev\std10042.spe
Start: 21.07.2014 18:19:11
Stop: 21.07.2014 18:20:51
BESSY: 298.6224845 298.2342694
IONI: 3.025313E-11 2.967582E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 77088
Run: 100000
Dead: 23
ICR: 71610
OCR: 0
Accumulated BESSY: 59666.6884675
Accumulated Ioni: 5.9704974E-09
N Accumulation: 200
Normfactor:  4.3454295e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mo
exp_x 71.2603988647461 71.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  42   2      2.293     28700.       549.      20.54
