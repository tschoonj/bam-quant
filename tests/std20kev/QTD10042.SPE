$DATA:
0 2000
17924
11460
5189
1636
386
75
22
16
10
7
3
5
5
0
4
13
196
1729
2243
2064
2026
2010
1948
1868
1768
1693
1813
1731
1642
1628
1626
1615
1575
1634
1585
1540
1558
1599
1533
1476
1492
1491
1554
1556
1561
1575
1569
1596
1561
1526
1517
1429
1412
1461
1371
1335
1270
1265
1147
1158
1108
1071
1009
982
910
969
902
872
785
761
811
742
788
747
730
746
739
694
645
627
593
577
566
574
558
519
544
572
520
522
557
524
555
532
540
513
510
564
583
558
640
647
612
560
606
570
585
623
804
1165
1891
2696
3750
4640
4801
4530
4291
3916
3726
3359
3036
2494
2170
1630
1314
987
794
622
578
548
446
474
448
392
373
364
395
396
389
348
393
432
375
374
380
408
426
403
421
363
364
341
326
315
318
333
312
321
324
320
330
314
304
336
313
296
334
276
285
294
295
308
297
341
328
280
299
264
291
291
309
312
296
290
307
277
280
290
264
306
299
301
307
301
310
273
293
292
291
309
260
277
291
271
286
274
289
280
253
283
261
267
257
278
271
245
249
268
277
290
289
286
277
262
271
284
291
236
295
257
278
267
273
271
277
262
240
245
233
266
254
253
267
256
261
249
267
259
243
249
225
254
270
281
273
260
252
231
239
275
241
264
276
249
261
246
263
284
286
284
282
289
263
231
275
243
276
234
266
216
269
192
254
261
261
237
254
260
251
241
289
241
253
256
245
227
259
224
280
214
244
249
261
221
237
229
252
234
253
248
265
228
277
272
347
355
357
383
357
339
339
293
261
281
269
264
238
252
232
236
228
234
216
261
231
216
230
239
234
239
248
218
228
223
223
275
241
275
240
245
288
234
248
229
259
245
257
230
251
236
209
259
236
241
225
269
220
242
226
235
253
210
238
238
217
224
243
217
263
225
224
234
222
209
218
220
216
228
235
219
230
236
235
225
224
208
243
239
233
237
232
225
252
194
248
225
263
229
249
226
265
254
239
242
241
250
243
231
241
251
232
227
207
223
217
241
228
240
255
232
224
217
247
244
214
219
219
247
228
241
242
250
258
220
234
217
228
204
232
224
231
249
234
205
225
231
236
233
260
250
280
257
233
240
228
247
229
220
206
232
235
212
218
249
218
245
234
265
259
251
237
252
256
249
233
209
267
215
246
239
211
239
233
244
263
229
236
233
218
229
219
230
235
233
237
258
243
233
242
186
236
239
261
249
202
198
247
206
265
254
248
262
258
247
245
248
220
249
238
239
242
247
250
238
228
226
238
222
227
229
256
200
214
218
226
229
235
222
225
245
240
250
252
263
222
259
248
229
271
257
270
243
229
254
245
252
256
214
253
286
227
251
259
237
233
258
248
267
240
247
243
243
260
233
261
265
253
267
243
251
256
259
253
273
245
270
247
245
238
250
272
246
269
260
260
255
263
266
266
268
292
256
272
258
263
263
281
257
317
269
274
254
293
270
294
318
301
279
270
307
287
305
308
286
292
282
260
280
289
311
276
293
299
303
280
307
278
317
262
308
293
282
310
289
311
297
315
300
320
293
293
281
309
323
289
322
299
309
346
354
345
322
317
352
349
325
319
318
319
300
367
320
341
314
340
326
339
325
349
360
361
347
348
364
372
389
348
357
386
392
330
395
412
391
392
421
396
396
382
359
387
393
408
415
405
388
431
451
404
406
425
411
403
434
433
424
430
479
439
450
401
421
463
413
473
469
463
428
461
472
451
447
465
475
463
461
478
489
464
483
514
500
488
474
525
517
506
502
508
499
559
488
529
560
604
613
610
673
679
757
877
1008
1140
1263
1400
1728
1819
1905
1953
2129
2111
1986
1878
1711
1570
1521
1369
1137
1076
974
907
936
914
910
951
934
925
991
1031
1117
1107
1121
1176
1175
1240
1231
1375
1300
1363
1463
1581
1554
1582
1589
1583
1701
1666
1720
1782
1751
1834
1748
1896
1941
1957
1969
1978
2069
2132
2157
2193
2282
2265
2443
2447
2654
2612
2736
2819
3060
3168
3425
3753
4023
4508
5396
6412
8146
10913
14793
21085
29349
41938
58658
80140
106022
136409
170940
207097
243208
275754
307193
328540
340596
342422
330616
309785
278561
241800
200563
160268
123194
90278
63354
43009
27960
17668
10746
6477
3960
2413
1598
1097
875
775
692
611
577
561
559
505
502
515
456
446
469
429
423
468
434
393
428
430
474
472
425
463
441
447
437
456
480
516
484
509
468
501
485
502
441
498
474
479
538
534
537
491
528
537
486
562
592
556
571
596
607
608
607
726
700
759
754
711
773
771
863
844
875
920
972
1019
1093
1194
1315
1580
1701
2262
2876
3851
5380
7353
10277
13816
18011
23613
29654
35700
40955
45954
48897
49882
49043
46189
42109
36814
30882
25360
19779
15220
11791
9403
7707
6945
6787
6890
7382
7553
7833
8064
7774
7205
6534
5645
4754
3728
2957
2034
1542
1003
641
429
301
183
121
70
40
41
26
26
22
27
28
12
15
14
19
16
17
11
11
18
7
15
8
8
14
6
10
14
11
9
5
8
8
9
14
8
4
6
7
10
10
10
5
10
6
3
6
7
7
6
8
9
7
6
7
10
3
5
3
6
7
8
2
7
6
4
4
6
8
7
4
5
6
10
6
10
8
11
8
12
13
14
5
13
7
19
19
12
9
8
14
6
7
10
6
6
6
10
3
3
7
7
3
10
7
3
6
7
5
6
4
1
3
4
3
6
6
2
4
2
2
6
3
8
2
6
1
7
4
0
2
6
7
4
1
2
3
4
6
7
5
4
4
2
5
5
3
2
2
2
2
2
7
4
3
5
7
4
2
5
1
6
2
2
4
3
3
7
1
5
1
4
4
4
3
6
5
3
8
2
4
4
3
4
4
2
6
4
7
3
5
2
7
3
4
3
1
2
4
3
8
3
3
3
2
1
1
2
4
0
3
6
4
3
4
2
2
5
5
4
3
3
1
3
1
5
2
1
3
6
2
5
1
1
2
4
2
1
0
2
4
8
1
2
0
7
1
2
2
5
3
2
4
2
4
2
2
2
3
4
6
5
5
7
2
5
4
3
1
5
0
2
1
3
1
0
5
3
3
3
4
2
4
5
2
3
5
3
1
5
5
4
3
2
5
4
1
5
4
4
0
4
3
1
2
5
0
2
1
3
2
3
0
1
5
3
3
0
2
6
4
2
3
8
1
5
2
4
4
4
5
1
5
1
2
0
4
5
3
3
3
2
3
4
2
1
5
3
3
3
3
3
2
6
1
4
2
4
2
4
0
4
2
1
4
4
1
2
3
5
4
4
2
5
2
3
3
3
3
7
0
1
1
1
2
2
2
2
5
2
5
3
1
1
4
2
1
4
4
5
1
3
2
1
3
2
2
3
1
1
3
4
2
1
4
2
2
1
4
2
2
3
3
3
1
1
2
3
4
1
1
3
4
2
2
5
4
2
4
4
2
8
5
4
2
1
3
2
1
0
1
0
1
1
4
1
4
6
3
2
2
2
3
3
4
3
2
6
4
2
2
2
2
1
6
4
2
4
3
2
6
2
3
7
4
4
3
5
2
1
6
1
3
3
5
6
3
3
2
5
5
1
1
2
2
4
4
1
2
9
5
4
5
4
3
3
3
2
3
2
3
2
4
7
4
4
3
5
1
4
5
6
7
4
3
3
2
0
3
4
5
3
9
5
4
4
3
7
8
4
0
2
3
2
1
4
5
4
4
1
6
4
2
2
3
4
2
5
4
7
3
3
2
6
3
4
7
1
7
4
8
8
3
3
3
6
5
5
5
6
3
1
4
4
1
12
7
8
7
5
7
5
5
3
6
3
6
6
4
5
7
8
7
4
6
5
3
7
5
3
5
10
2
8
8
8
4
7
13
13
10
18
9
7
25
10
15
15
18
20
20
18
22
20
20
14
9
11
18
17
10
3
14
14
9
9
14
8
14
15
11
13
14
12
19
6
13
17
13
17
18
27
19
19
23
21
26
13
20
16
25
11
20
24
27
25
27
20
24
28
27
21
31
29
31
25
40
47
40
50
61
67
72
96
125
174
188
208
252
327
378
477
541
627
743
870
929
1088
1157
1134
1245
1287
1420
1395
1268
1342
1270
1251
1152
973
946
860
731
634
524
456
328
257
220
154
143
88
61
50
33
17
24
20
14
15
7
2
6
6
6
3
6
5
4
6
12
10
12
4
8
8
7
12
4
5
9
7
5
8
10
3
4
12
8
7
10
13
7
11
12
12
7
12
8
11
11
16
5
8
11
18
11
13
4
10
12
11
10
16
22
19
29
23
24
29
39
50
56
54
65
92
124
122
155
167
175
233
249
287
316
319
333
411
382
411
388
383
369
426
355
288
301
285
238
234
180
148
152
117
106
110
115
77
77
76
65
65
46
48
50
32
20
30
25
20
16
13
9
16
3
4
1
10
2
2
2
1
0
1
0
1
0
0
0
0
3
0
0
0
0
2
1
1
1
1
1
0
2
1
0
0
2
0
1
1
0
1
1
1
1
2
3
2
2
3
2
1
2
2
4
3
5
2
4
4
11
7
4
8
10
9
13
16
15
22
18
20
22
34
29
24
25
28
33
27
27
28
17
18
17
14
18
11
13
9
8
11
11
10
10
10
11
10
6
6
6
4
6
2
7
2
2
0
2
2
2
0
0
0
0
0
0
1
0
1
0
1
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10042.spe
Start: 21.07.2014 18:19:11
Stop: 21.07.2014 18:20:51
BESSY: 298.6224845 298.2342694
IONI: 3.025313E-11 2.967582E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 77088
Run: 100000
Dead: 23
ICR: 71610
OCR: 0
Accumulated BESSY: 59666.6884675
Accumulated Ioni: 5.9704974E-09
N Accumulation: 200
Normfactor:  4.3454295e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mo
 
exp_x 71.2603988647461 71.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  42   2      2.293     28700.       549.      20.54
