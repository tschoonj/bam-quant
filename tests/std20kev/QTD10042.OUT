     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:46  บ   
     บ  Spectrum: QTD10042.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        2 iterations done    บ   
     บ  ChiSquare =  3010.2   last change =   .01%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -18.0 ๑   100.0   บ     -18.0 ๑     9.9   บ   
     บ  GAIN (eV/ch)  บ     20.073 ๑   2.000   บ    20.073 ๑    .087   บ   
     บ det NOISE (eV) บ      104.1 ๑    40.0   บ     104.1 ๑    18.3   บ   
     บ  FANO factor   บ       .212 ๑    .050   บ      .212 ๑    .054   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Mo-L1                              2448. ๑    204.           บ   
     บ    L1M3        2.468    .76664        1877. ๑    171.      8.7  บ   
     บ              123.846    148.38        4770.           2.64E-01  บ   
     บ    L1N3        2.831    .23336         571. ๑     83.      3.8  บ   
     บ              141.930    153.82        4582.           4.02E-01  บ   
     บ                                                                 บ   
     บ  2 Mo-L2                             16618. ๑    625.           บ   
     บ    L2M4        2.395    .90817       15092. ๑    572.     17.1  บ   
     บ              120.210    147.26        4802.           2.35E-01  บ   
     บ    L2M1        2.120    .02827         470. ๑     70.     68.0  บ   
     บ              106.509    142.97        4593.           1.46E-01  บ   
     บ    L2N1        2.624    .06355        1056. ๑     79.      3.1  บ   
     บ              131.618    150.74        4685.           3.28E-01  บ   
     บ                                                                 บ   
     บ  3 Mo-L3                             29886. ๑    572.           บ   
     บ    L3M5        2.293    .96031       28700. ๑    553.     20.5  บ   
     บ              115.128    145.68        4855.           1.94E-01  บ   
     บ    L3M1        2.061    .02802         837. ๑     62.    198.1  บ   
     บ              103.570    142.03        3545.           1.24E-01  บ   
     บ    L3N1        2.456    .01168         349. ๑     69.      8.7  บ   
     บ              123.248    148.20        4770.           2.59E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                              35. ๑     68.           บ   
     บ    L1N5        4.586    .64875          23. ๑     79.       .9  บ   
     บ              229.362    177.79        4264.           7.93E-01  บ   
     บ    L1N4        4.711    .35125          12. ๑     69.      1.2  บ   
     บ              235.591    179.37        4203.           8.08E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
